src/dotnet/UI.Blazor.App/Components/ChatView/Items/LinkPreview/LocalLinkPreview.razor

```diff
@@ -49,22 +42,57 @@
                 <a class="c-description" href="@url" @onclick:preventDefault="true" @onclick="NavigateToUrl">@entry.Content</a>
             </div>
         }
+
+        @if (entry is not null && entry.Attachments.Count > 0) {
```

Comment:
Do we really want to show attachmments inside local link preview?
Just mention that there are attachments is not enough?

Reply:
Discussed with Alex K.